config/obsoletion.yml

```diff
@@ -12,3 +12,20 @@ changed_parameters:
     parameters: IgnoredPatterns
     alternative: AllowedPatterns
     severity: warning
+
+renamed:
```

Comment:
[Explanation why this is a breaking change](https://github.com/rubocop/rubocop/pull/10725#issuecomment-1163565021)